packages/desktop-client/src/components/accounts/Account.tsx

```diff
@@ -781,7 +782,9 @@ class AccountInternal extends PureComponent<
       const account = this.props.accounts.find(
         account => account.id === this.props.accountId,
       );
-      this.props.dispatch(updateAccount({ ...account, name } as AccountEntity));
+      this.props.dispatch(
+        updateAccount({ account: { ...account, name } as AccountEntity }),
```

Comment:
Since we're already here, wondering why this cast is needed? Isn't `this.props.accounts` already an `AccountEntity[]`?

Reply:
Nice catch :) Pushed an update, TS just needed a hint that the account variable is not undefined.